src/Sulu/Bundle/ContactBundle/Resources/js/components/ContactDetails/ContactDetails.js

```diff
@@ -173,9 +172,8 @@ class ContactDetails extends React.Component<Props> {
         const {value: {phones}} = this.props;
 
         if (phones.length === 0) {
-            // $FlowFixMe
             return [
-                this.getEmptyEntry('phones'),
+                (this.getEmptyEntry('phones'): any),
```

Comment:
I would not just cast it to `any`, since we are losing potential subsequent type checks. If we do something like this, I would cast to the type we want afterwards as well:

```javascript
((this...): any): PhoneType)
```